src/yoroi-wallets/hooks/index.ts

```diff
@@ -119,6 +118,11 @@ export const useUtxos = (wallet: YoroiWallet) => {
   return wallet.utxos
 }
 
+export const useAssetIds = (wallet: YoroiWallet): string[] => {
+  const utxos = useUtxos(wallet)
+  return utxos.flatMap((utxo) => utxo.assets).map((asset) => asset.assetId)
```

Comment:
shouldnt it transform to what Yoroi understands as tokenId? (merging YoroiNFT -> TokenInfo), assets or native assets are Cardano things.

Reply:
my point here was more to don't consume `utxo` directly, any reason why not to use `useBalances / Amounts.toArray ?`

Reply:
you can return string TokenID will be removed